client/src/pages/organizations/Show.js

```diff
@@ -373,6 +376,22 @@ const OrganizationShow = ({ pageDispatchers }) => {
                   humanValue={Organization.humanNameOfType}
                 />
 
+                {renderLeadingPositions(
+                  organization.positions,
+                  PositionRole.LEADER.toString(),
+                  pluralize(
+                    utils.titleCase(PositionRole.LEADER.humanNameOfRole())
+                  )
+                )}
+
+                {renderLeadingPositions(
+                  organization.positions,
+                  PositionRole.DEPUTY.toString(),
+                  pluralize(
+                    utils.titleCase(PositionRole.DEPUTY.humanNameOfRole())
+                  )
+                )}
```

Comment:
This probably warrants extra wdio tests; I imagine you could create `client/tests/webdriver/baseSpecs/showOrganization.spec.js` (with an accompanying page object), which tests some conditions, like for EF 2 these two fields should not be shown, and for EF 2.2 they should show the current leader and the current deputy.

Reply:
Applied